src/main/java/uk/gov/justice/digital/delius/data/api/OffenderAssessments.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.delius.data.api;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class OffenderAssessments {
+    private int rsrScore;
```

Comment:
Maybe add some swagger explaining what an rsrScore is (we haven't been very good at that but be good to improve things with new fields)

Reply:
done